crates/mvr-cli/src/utils/sui_binary.rs

```diff
@@ -81,12 +83,28 @@ pub fn force_build() -> Result<(), Error> {
     Ok(())
 }
 
-fn sui_command(args: Vec<&str>) -> Result<Output, Error> {
+/// Gets the active network by calling `sui client chain-identifier` from the Sui CLI.
+/// Returns the network as a `Network` enum, or errors if network is not `mainnet` or `testnet`.
+pub fn get_active_network() -> Result<Network, Error> {
+    let output = sui_command(["client", "chain-identifier", "--json"].to_vec())?;
+
+    let output = String::from_utf8_lossy(&output.stdout).to_string();
+
+    let chain_id: String = serde_json::from_str(&output).map_err(|_| {
+        CliError::UnexpectedParsingError(
+            "Failed to parse chain identifier using the SUI binary.".to_string(),
+        )
+    })?;
```

Comment:
Do you really need to parse it back from JSON? Could you instead do `client chain-identifier`, and set the `stdout` to Stdin::piped() in the `sui_command`, and then you wouldn't have to do serde_json::from_str. 
```suggestion
    let chain_id_str = String::from_utf8_lossy(&output.stdout).to_string();
    
```

Reply:
Yeah not necessary indeed. It just missed a "trim()"!